test/e2e/serial/tests/non_regression.go

```diff
@@ -283,7 +284,8 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			nrtPostCreate, err := e2enrt.FindFromList(nrtPostCreateDeploymentList.Items, updatedPod.Spec.NodeName)
 			Expect(err).ToNot(HaveOccurred())
 
-			policyFuncs := tmPolicyFuncsHandler[nrtv1alpha2.TopologyManagerPolicy(nrtInitial.TopologyPolicies[0])]
+			nrtInitialTopologyPolicies := noderesourcetopology.GetTopologyPolicies(*nrtInitial)
+			policyFuncs := tmPolicyFuncsHandler[nrtv1alpha2.TopologyManagerPolicy(nrtInitialTopologyPolicies[0])]
```

Comment:
again we should just pick the first attribute, possibly erroring out of there are more attributes